### PR TITLE
Script to generate keystore

### PR DESCRIPTION
**Pull request by nomaxg**
Simple script to generate a new keystore file from a password. Necessary because we wish to use a real L1 wallet for deployment (not the test test junk mnemonic).

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -31,6 +31,11 @@ ESPRESSO_ZKEVM_FAUCET_NUM_CLIENTS=1
 ESPRESSO_ZKEVM_FAUCET_TRANSACTION_TIMEOUT_SECS=300
 ESPRESSO_ZKEVM_IGNORE_GEN_BLOCK_NUMBER_CHECK=1
 
+# Keystore information for the zkevm node
+ESPRESSO_ZKEVM_KEYSTORE_PASSWORD=testonly
+ESPRESSO_ZKEVM_KEYSTORE_DIR=/zkevm-node/test
```
Comment (jbearer) [inline .env:36]: Why do we use an absolute path here?

File: keygen/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    #[clap(
+        long,
+        env = "ESPRESSO_ZKEVM_KEYSTORE_PASSWORD",
+        default_value = "testonly"
+    )]
+    password: String,
+
+    #[clap(
+        long,
+        env = "ESPRESSO_ZKEVM_KEYSTORE_PASSWORD",
```
Comment (jbearer) [inline keygen/src/main.rs:20]: Should be `ESPRESSO_ZKEVM_KEYSTORE_NAME`?

**Review by jbearer — APPROVED** [review]
LGTM, I think merging with main (where Mat made changes for open sourcing) should fix the CI

**Comment by nomaxg** [comment]
> 

The build is currently broken, working on fixing here: https://github.com/EspressoSystems/espresso-polygon-zkevm-demo/pull/125

**Pull request merged**